cluster/update-storage-objects.sh

```diff
@@ -59,7 +61,12 @@ for resource in "${resources[@]}"
 do
   for namespace in "${namespaces[@]}"
   do
+    # If get fails, assume it's because the resource hasn't been installed in the apiserver.
+    # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
+    # resources.
+    set +e
```

Comment:
I was thinking that consider "the server doesn't have a resource type" as an error as well.
Can we get that the command exited with exit 0 or exit 1?
There should be atleast one resource for which the command exited with  "exit 0".